x-pack/plugins/security_solution/server/lib/detection_engine/rules/find_rules.ts

```diff
@@ -8,13 +8,23 @@
 import { FindResult } from '../../../../../alerting/server';
 import { SIGNALS_ID } from '../../../../common/constants';
 import { RuleParams } from '../schemas/rule_schemas';
+import { ruleTypeMappings } from '../signals/utils';
 import { FindRuleOptions } from './types';
 
-export const getFilter = (filter: string | null | undefined) => {
+export const getFilter = (
+  filter: string | null | undefined,
+  isRuleRegistryEnabled: boolean = false
```

Comment:
`isRuleRegistryEnabled` needs to be passed in on line 45 below to make the new rule types show up in the rules table

Reply:
Ergh, sorry about that! Pushed up the change.